packages/help-extension/src/index.tsx

```diff
@@ -341,7 +341,7 @@ function activate(
       );
       const copyright = (
         <span className="jp-About-copyright">
-          {trans.__('© 2015-2020 Project Jupyter Contributors')}
+          {trans.__('© 2015-2021 Project Jupyter Contributors')}
```

Comment:
How about

```tsx
{trans.__('© 2015-%1 Project Jupyter Contributors', new Date().getFullYear())}
```

Reply:
Sure. Let's do it as a suggestion so we can commit from the GitHub UI.

```suggestion
          {trans.__('© 2015-%1 Project Jupyter Contributors', new Date().getFullYear())}
```